api/routes/middlewares/index.js

```diff
@@ -50,6 +50,10 @@ import passport from 'passport';
 middlewares.use(passport.initialize());
 middlewares.use(passport.session());
 
+// rate limiter middleware to reduce API calls to server within time frame
+import rateLimiter from 'shared/middlewares/rateLimiter';
+middlewares.use(rateLimiter('api', 2500, 3600000));
```

Comment:
outsider pov: 
imo bad (aka magic) numbers b/c i need to look at source what they mean 

`rateLimiter('api', MAX_REQUESTS, ONE_HOUR)`

normally i'd say only name constants and variables by WHAT not HOW but i think stuff like ONE_HOUR is an exception b/c you would do it with syntax like `1.hour` if you could - so it's more a workaround to make it readable in JS

Reply:
Yeah totally, I don't think @LetItRock had thought those numbers through yet

Reply:
@andreasklinger yeah for sure I will extract those values to more readable constants... till today I didn't know what exact values they gonna be... and we are still debating on values for hyperion...